Module 5/Problem 27/problem27.json

```diff
@@ -0,0 +1,86 @@
+Input:
+
+[
+  {
+    "returnId": "10051",
+    "returnQuantity": 2,
+    "returnPrice": 100,
+    "returnTax": 12.56,
+    "country": "USA"
+  },
+  {
+    "returnId": "10052",
+    "returnQuantity": 25,
+    "returnPrice": 35,
+    "returnTax": 29.85,
+    "country": "CA"
+  },
+  {
+    "returnId": "10053",
+    "returnQuantity": 5,
+    "returnPrice": 55,
+    "returnTax": 9.85,
+    "country": ""
+  }
+]
+
+
+Transformation:
+
+
+[
+  {
+    "operation": "modify-overwrite-beta",
+    "spec": {
+      "*": {
+        "temp": "=divide(1,@(1,returnQuantity))",
+        "Return Amount": "=divideAndRound(2,@(1,returnPrice),@(1,temp))",
+        "Return Total Amount": "=doubleSum(@(1,Return Amount),@(1,returnTax))",
+        "ReturnId": "@(1,returnId)"
+      },
+      "[0]": {
+        "Country": "United States od America"
+      },
+      "[1]": {
+        "Country": "Canada"
+      },
+      "[2]": {
+        "Country": "NA"
```

Comment:
Indexing is hard coded. Make sure your approach is always loosely written, so that even if there are more fields you don't need to change the approach.